docs/pages/admin-guides/teleport-policy/database-access-controls.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: Teleport Policy and Database Access Controls
+description: Describes how to import and visualize advanced Teleport Database Access Controls policies using Teleport Policy and Access Graph.
+---
+
+Advanced Database Access Controls features allow Teleport administrators to configure user permissions for individual database objects using flexible, label-based approach for expressing access policies.
+
+The database objects-level access information is automatically synchronized to Teleport Policy, making it possible to see who has particular levels of access to the different parts of the database.
+
+<Notice type="note">
+Currently PostgreSQL is the only supported protocol for database-object level permissions. Support for additional protocols will be added in the future.
+</Notice>
+
+## How it works
+
+Teleport continuously scans connected databases and imports objects, such as tables, based on predefined import rules. These rules determine which objects to import and what labels to assign to them. By default, all objects are imported, with labels automatically generated from object properties like schema and database name.
+
+The labels assigned to these objects are matched against the `spec.allow.db_permissions` section of each user role. This results in a set of permissions that will be applied when the user connects to the database.
+
+For more details of this process see [Database Access Controls](../../enroll-resources/database-access/rbac.mdx).
```

Comment:
nit. I would remove this line. This page already provides a good example of how to use TAG.  Readers can find references at the end of the page too. 
```suggestion
```